test/controllers/presenter.controller.spec.ts

```diff
@@ -54,13 +54,20 @@ describe("PRESENTER controller", () => {
   });
 
   describe("POST tests", () => {
-    test("redirect the entity page after a succesful post from presenter page", async () => {
+    test("redirect the entity page after a successful post from presenter page", async () => {
       const resp = await request(app).post(PRESENTER_URL).send(PRESENTER_OBJECT_MOCK);
 
       expect(resp.status).toEqual(302);
       expect(resp.text).toContain(ENTITY_PAGE_REDIRECT);
     });
 
+    test("redirect the entity page after a successful post from presenter page with special characters", async () => {
```

Comment:
Could name this 'redirect to the entity page...' here and above, just to make English a bit clearer.

Reply:
Must say we're getting into some really fine detail here, I have updated it.